models/GameCards.js

```diff
@@ -0,0 +1,21 @@
+import mongoose from "mongoose";
+
+const { Schema } = mongoose;
+const gameCardSchema = new Schema({
+  id: { type: String, required: false },
```

Comment:
you dont need to define an extra id here, mongoDB does that for you

Reply:
Hey Mark, thanks for the input. Deleted the id in the new schema